contracts/utils/VaultQueue.sol

```diff
@@ -43,6 +42,7 @@ contract VaultQueue is
 
     address public wethVault;
     address public stethVault;
```

Comment:
These need to be immutable variables that are set on constructor

Reply:
You can't use immutable in `initialize()` and can't use their constructors with upgradeable contracts.

https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable

>You can use your Solidity contracts with OpenZeppelin Upgrades without any modifications, except for their constructors